hw/ip/otbn/data/otbn_testplan.hjson

```diff
@@ -3,30 +3,68 @@
 // SPDX-License-Identifier: Apache-2.0
 {
   name: "otbn"
-  // TODO: remove the common testplans if not applicable
   import_testplans: ["hw/dv/tools/dvsim/testplans/csr_testplan.hjson",
                      "hw/dv/tools/dvsim/testplans/mem_testplan.hjson",
                      "hw/dv/tools/dvsim/testplans/intr_test_testplan.hjson",
                      "hw/dv/tools/dvsim/testplans/tl_device_access_types_testplan.hjson"]
   entries: [
     {
-      name: sanity
+      name: smoke
       desc: '''
-            Basic sanity test acessing a major datapath within the otbn.
+            Smoke test, running a single fixed binary
 
-            **Stimulus**:
-            - TBD
+            This runs the binary from otbn/dv/smoke/smoke_test.s, which is
+            designed to check most of the implemented instructions.
```

Comment:
add "in a predictable manner" maybe?

Having a smoke test which is fully predictable is really helpful for (PR) CI runs, where we need to reduce the chance of unrelated failures as much as possible.

Reply:
Well, we'll set a seed in the CI runs, so I'd hope they will be predictable either way. But that's a good point: I'll update this.